kotlin/workflow-core/src/main/java/com/squareup/workflow/RenderContext.kt

```diff
@@ -178,7 +178,7 @@ fun <StateT, OutputT : Any> RenderContext<StateT, OutputT>.runningWorker(
  */
 fun <EventT, StateT, OutputT : Any> RenderContext<StateT, OutputT>.makeEventSink(
```

Comment:
Could you add `Sink.asEventSink` and make this wrap that? And deprecate this.

Too much?

Reply:
Oo, I like.

Reply:
Actually, not sure. It would have to be an extension on `Sink<WorkflowAction<S, O>>`, and I'm not sure what the use case for having one of those that is not `actionSink` would be. I'm gonna leave it out for now.